src/main/java/com/github/timm/cucumber/generate/ClassNameGenerator.java

```diff
@@ -1,37 +1,38 @@
-package com.github.timm.cucumber.generate;
```

Comment:
Sorry for that mistake. I am using Intellij IDEA 14.1.1 . will fix this as soon as possible.